scripts/run_lint.sh

```diff
@@ -21,14 +21,12 @@ function display_result {
 flake8 --max-complexity 10 --count
 display_result $? 1 "Flake 8 code style check"
 
-pylint --rcfile=.pylintrc -j 0 ./app ./tests
+pylint --reports=n --output-format=colorized --rcfile=.pylintrc -j 0 ./app ./tests
```

Comment:
we only get errors or warnings output to console now; not the full code-tree and tabulated results etc. errors and warnings are coloured coded